### PR TITLE
add bool isBirthday to formatted data returned by getFriendList() and …

### PR DESCRIPTION
**Pull request by RandallKent**
…getUserInfo()

### CODE REVIEW

**Comment by Schmavery** [comment]
Can you add isBirthday to the docs where we list what is returned by [getFriendsList()](https://github.com/Schmavery/facebook-chat-api/blob/master/DOCS.md#apigetfriendslistcallback)?

Also, if you don't mind, update `is_friend, is_birthday` to `isFriend, isBirthday` in the docs for [getUserInfo()](https://github.com/Schmavery/facebook-chat-api/blob/master/DOCS.md#apigetuserinfoids-callback), seeing as that appears to be out of date.

**Comment by Schmavery** [comment]
Looks good to me.  @bsansouci feel free to merge if you agree.

**Comment by bsansouci** [comment]
I updated the tests, and `isFriend` can you update `isBirthday` to follow that? Thanks a lot.

File: src/getFriendsList.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ function formatData(obj) {
       type: user.type,
       profileUrl: user.uri,
       vanity: user.vanity,
+      isBirthday: (user.is_birthday) ? true : false,
```
Comment (bsansouci) [inline src/getFriendsList.js]: Can you update this to be `!!user.is_birthday`

**Comment by bsansouci** [comment]
I think we're good to merge once you update that little thing :D

**Comment by RandallKent** [comment]
Boom!

**Comment by bsansouci** [comment]
Wooop. Thanks @RandallKent :)

**Pull request merged**